1558751271601	Sync.LogManager	DEBUG	Flushing file log
1558751271613	Sync.LogManager	DEBUG	Log cleanup threshold time: 1557887271613
1558751271691	Sync.LogManager	DEBUG	Done deleting files.
1558751272568	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558751274058	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558751281723	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558751320469	Sync.RemoteTabs	INFO	Generating tab list with filter
1558751320469	Sync.RemoteTabs	INFO	Final tab list has 0 clients with 0 tabs.
1558751587378	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1558751587378	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558751587381	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1558751588378	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558751589628	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1558751589628	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1558751589634	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Linux x86_64) FxSync/1.69.0.20190521202118.desktop
1558751589634	Sync.Service	INFO	Starting sync at 2019-05-24 20:33:09 in browser session p3tpBcsAkZXJ
1558751589635	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558751589639	Sync.Status	INFO	Resetting Status.
1558751589639	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1558751589651	Sync.Resource	WARN	GET request to https://sync-548-us-west-2.sync.services.mozilla.com/1.5/84144729/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1558751589652	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1558751589652	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1558751589652	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1558751589653	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1558751589656	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
